client/lxd_instances.go

```diff
@@ -609,6 +612,8 @@ func (r *ProtocolLXD) CreateInstance(instance api.InstancesPost) (Operation, err
 	return op, nil
 }
 
+// tryCreateInstance attempts to create a new instance on multiple target servers specified by their URLs.
```

Comment:
Please can you split the last 2 comments into their own commit to aid backporting.